app/Models/Permission.php

```diff
@@ -30,6 +30,4 @@
  * @method static \Illuminate\Database\Eloquent\Builder|Permission whereUpdatedAt($value)
  * @mixin \Eloquent
  */
-class Permission extends \Spatie\Permission\Models\Permission
-{
-}
+class Permission extends \Spatie\Permission\Models\Permission {}
```

Comment:
We don't need to do this anymore - because we use enums for permissions. This was only because we wanted to have constants on these models.